openc3/lib/openc3/interfaces/interface.rb

```diff
@@ -133,6 +139,8 @@ def initialize
       @name = self.class.to_s.split("::")[-1] # Remove namespacing if present
       @state = 'DISCONNECTED'
       @target_names = []
```

Comment:
We're keeping @target_names for now or is this deprecated?

Reply:
Keeping it for backwards compatibility.  Its a superset of cmd_target_names and tlm_target_names.   Generally only the newer lists should be used, but if you just want to know all targets supported, target_names still saves you having to merge the other two lists and uniq.